client/js/components/image-viewer.js

```diff
@@ -48,21 +56,35 @@ const createImageViewer = (viewer) => {
 
     image.addEventListener('dblclick', (e) => {
       setupViewer(imageInfoSrc, viewer, viewerId);
-      enterFullscreen(viewerContent);
-      trackGaEvent({
+      const gaData = {
         category: 'component',
         action: 'work-enter-fullscreen-image:dblclick',
         label: `id:${window.location.pathname.match(/\/works\/(.+)/)[1]}, title:${document.title}`
-      });
+      };
+      if (hasFullscreen()) {
+        enterFullscreen(viewerContent);
+        trackGaEvent(gaData);
```

Comment:
`trackGaEvent(gaData)` could live after the `if/esle` because it's getting called from both condition?